docs/source/_toctree.yml

```
@@ -36,8 +36,23 @@
           title: Overview
         - local: clickhouse
           title: ClickHouse
-        - local: duckdb
-          title: DuckDB
+        - title: DuckDB
```

didn't work, reverting
```suggestion
        - local: duckdb
          title: DuckDB
```